src/EditorFeatures/Core/Shared/IDocumentSupportsFeatureService.cs

```diff
@@ -1,14 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using Microsoft.CodeAnalysis.Host;
-
-namespace Microsoft.CodeAnalysis.Editor.Shared
-{
-    internal interface IDocumentSupportsFeatureService : IWorkspaceService
-    {
-        bool SupportsCodeFixes(Document document);
-        bool SupportsRefactorings(Document document);
-        bool SupportsRename(Document document);
-        bool SupportsNavigationToAnyPosition(Document document);
-    }
-}
```

Comment:
any IVT compat concerns here?

Reply:
I was not able to find any obvious usages of this service in other languages

Tagging @heejaechang to confirm

Reply:
This change breaks VS for Mac. Looking into how best to fix.

Reply:
@avodovnik You just need to remove this file: https://github.com/mono/monodevelop/blob/master/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.RoslynServices/MonoDevelopDocumentSupportsFeatureService.cs